.buildkite/deploy/kubernetes/deploy-k8s-pipeline.yml

```diff
@@ -10,7 +10,10 @@ steps:
 
     steps:
     - label: "Checks"
-      command: ".buildkite/deploy/kubernetes/scripts/make.sh"
+      command: |
+        set -euo pipefail
+        make -C deploy/kubernetes all
+        make check-no-changes
```

Comment:
leaving the comment here because I can't pin point to the exact line (line 23) but could we replace

`context: "Deploy/k8s-test v1.28.0"` with `context: "deploy/k8s test v1.28.0"` everywhere in this file? (correct capitalization helps with the reported status output and improves discoverability)